File: php/PetShop.php
<?php
class PetShop
{
    // atribut private
    private $id;
    private $name;
    private $price;
    private $stock;
    private $image;

    // konstruktor dengan parameter untuk mengisi nilai
    public function __construct($id, $name, $price, $stock, $image)
    {
        $this->id = $id;
        $this->name = $name;
        $this->price = $price;
        $this->stock = $stock;
        $this->image = $image;
    }

    // getter dan setter untuk id
    public function setId($id)
    {
        $this->id = $id;
    }

    public function getId()
    {
        return $this->id;
    }

    // getter dan setter untuk name
    public function setName($name)
    {
        $this->name = $name;
    }

    public function getName()
    {
        return $this->name;
    }

    // getter dan setter untuk price
    public function setPrice($price)
    {
        $this->price = $price;
    }

    public function getPrice()
    {
        return $this->price;
    }

    // getter dan setter untuk stock
    public function setStock($stock)
    {
        $this->stock = $stock;
    }

    public function getStock()
    {
        return $this->stock;
    }

    // getter dan setter untuk image
    public function setImage($image)
    {
        $this->image = $image;
    }

    public function getImage()
    {
        return $this->image;
    }

    function __destruct() {}
}

File: java/PetShop.java
class PetShop {

    // atribut-atribut private
    private int id;
    private String name;
    private int price;
    private int stock;

    // constructor dengan parameter
    PetShop(int id, String name, int price, int stock) {
        this.id = id;
        this.name = name;
        this.price = price;
        this.stock = stock;
    }

    // getter dan setter untuk id
    public void setId(int id) {
        this.id = id;
    }

    public int getId() {
        return id;
    }

    // getter dan setter untuk name
    public void setName(String name) {
        this.name = name;
    }

    public String getName() {
        return name;
    }

    // getter dan setter untuk price
    public void setPrice(int price) {
        this.price = price;
    }

    public int getPrice() {
        return price;
    }

    // getter dan setter untuk stock
    public void setStock(int stock) {
        this.stock = stock;
    }

    public int getStock() {
        return stock;
    }
}
